.github/workflows/ci.yaml

```diff
@@ -43,11 +43,15 @@ jobs:
       - name: Run build
         run: npm run compile
 
-      # Run tests
-      - name: Run test
+      - name: Run test (current vscode)
         uses: GabrielBB/xvfb-action@v1.0
         with:
-          run: npm test
+          run: npm run test
+
+      - name: Run test (oldest vscode)
```

Comment:
Separate tests should be in separate checks via a matrix. Otherwise, it makes it harder to navigate through the CI results.

Reply:
I really want to avoid an exploding matrix like we already have on the language server. Running the same tests with oldest vscode after and only if first fails makes sense and avoids creating an unmanageable list of individual jobs.

Reply:
Well, to me it's unmanageable to debug weird failures when one thing runs in an unclean env. Plus, it's impossible to tell what failed by looking at a number of jobs.